include/avif/avif.h

```diff
@@ -482,6 +482,7 @@ typedef struct avifRGBImage
     uint32_t rowBytes;
 } avifRGBImage;
 
+// Sets rgb->width, rgb->height, and rgb->depth to image->width, image->height, and image->depth. Sets rgb->pixels to NULL and rgb->rowBytes to 0. Sets the other fields of 'rgb' to default values.
```

Comment:
Joe: I forgot your preference for how long a comment line can be before it is folded.

Reply:
My editor breaks those comment lines like this:

```
// Sets rgb->width, rgb->height, and rgb->depth to image->width, image->height, and image->depth.
// Sets rgb->pixels to NULL and rgb->rowBytes to 0. Sets the other fields of 'rgb' to default
// values.
```

I believe it is 100 lines, although I can't legislate that in clang-format independent of other things, I don't believe.

Reply:
Thanks! Yes, it looks like a 100-char line length. clang-format did not fold that very long comment line. Maybe we can document the 100-char comment line length in README.md?